pages/common/git-authors.md

```diff
@@ -0,0 +1,17 @@
+# git authors
+
+> Display log of authors of a Git repository.
+> A part of git-extras.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-authors>.
+
+- Display log of authors/committers of a Git repository:
```

Comment:
I tried it; this command does a bit more:
```suggestion
- Create a new (or update the existing) `AUTHORS` file in the current directory, populate it from the current Git repository, and open it in the default editor:
```

Reply:
I really did wonder about that file, I thought it was the command editing the AUTHORS file, turns out it didn't exist!

Thanks for that